File: solution/1550. Three Consecutive Odds/1550.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int odds = 0;
        for (int x : arr) {
            odds = x % 2 ? odds + 1 : 0;
            if (odds == 3)
                return true;
        }
        return false;
    }
};

File: solution/1550. Three Consecutive Odds/1550.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        odds = 0
        for x in arr:
            odds = odds + 1 if x % 2 == 1 else 0
            if odds == 3:
                return True
        return False
    
